app/lib/scorecard/backend.dart

```diff
@@ -292,6 +308,31 @@ class ScoreCardBackend {
         dartdocReport: reports[ReportType.dartdoc] as DartdocReport,
       );
 
+      // store compressed report on the ScoreCard entity too
+      for (final scr in reportEntities) {
+        if (scr == null || scr.reportJsonGz == null) continue;
+        final size = scr.reportJsonGz.length;
+        if (size > _reportSizeDropThreshold) {
+          // TODO: replace it with meaningful content
+          _logger.shout(
+              '${scr.reportType} report exceeded size threshold ($size > $_reportSizeWarnThreshold)');
```

Comment:
Consider using something like:
```dart
extension CustomException on Logger {
  reportError(String message) =>
       this.shout(message, Exception(message), StackTrace.current);
}
```
So errors show up in stack driver error reporting.